doc/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = python3 -msphinx
+SPHINXBUILD   = python3.6 -msphinx
```

Comment:
Oops, this was my fault, needed to build the docs locally. Reminder to revert this before merging.

Reply:
```suggestion
SPHINXBUILD   = python3 -msphinx
```